tests/tasks/great_expectations/test_great_expectations_checkpoints.py

```diff
@@ -0,0 +1,78 @@
+from unittest.mock import MagicMock
+import prefect
+from prefect import context
+from prefect.tasks.great_expectations import RunGreatExpectationsCheckpoint
+from prefect.utilities.configuration import set_temporary_config
+import pytest
+import os
+import shutil
+import tempfile
+
+
+class TestInitialization:
+    def test_inits_with_no_args(self):
+        t = RunGreatExpectationsCheckpoint()
+        assert t
+
+    def test_kwargs_get_passed_to_task_init(self):
+        t = RunGreatExpectationsCheckpoint(
+            checkpoint_name="checkpoint",
+            context_root_dir="/path/to/somewhere",
+            runtime_environment={
+                "plugins_directory": "/path/to/plugins/somewhere/else"
+            },
+            run_name="woooo",
+        )
+        assert t.checkpoint_name == "checkpoint"
+        assert t.context_root_dir == "/path/to/somewhere"
+        assert t.runtime_environment == {
+            "plugins_directory": "/path/to/plugins/somewhere/else"
+        }
+        assert t.run_name == "woooo"
+
+    def test_raises_if_checkpoint_not_provided(self, monkeypatch):
+        task = RunGreatExpectationsCheckpoint()
+        client = MagicMock()
+        great_expectations = MagicMock(client=client)
+        monkeypatch.setattr("prefect.tasks.great_expectations", great_expectations)
+        with pytest.raises(ValueError, match="checkpoint"):
+            task.run()
+
+
+@pytest.fixture(scope="session")
+def example_context_root_dir():
+    """
+    Copies the examples configuration directory to the proper location for the test and exposes its path for the test.
+    """
+    with tempfile.TemporaryDirectory() as tmp:
+        examples_path = os.path.abspath(
+            "examples/task_library/great_expectations/great_expectations/"
+        )
+        shutil.copytree(examples_path, tmp, dirs_exist_ok=True)
+        yield tmp
+
+
+@pytest.mark.skipif("sys.version_info <= (3,8)")
+class TestIntegrationWithLocalConfig:
+    """
+    Integration test to check the task library task still integrates with how Great Expectations checkpoints are configured.
```

Comment:
This comment explains my reasoning I think; let me know if you disagree with the fundamental ideal here that the integration test, checking that Great Expectations still acts the way we think it does, can be run on 3.8 only.

Reply:
Makes sense to me! No problem with only running on 3.8